### DISCUSSION TITLE
Some vanilla event conversations appear on player who is not interacting with the event npc

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14293] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

some events that can be triggered by other player lack ClearTagAction
some ConversationActions does not have TargetTag="triggerer_player"

### Reproduction steps

1. check every events :(

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
This is very confusing and annoying and in rare situation if multiple players choose option in the same time the conversation will not end and close

**Comment by ubertpendragon** [comment]
add path unlocking events

If player1 refuses to pay and player2 triggers the conversation player1 will see it I think

**Comment by A2ur3** [comment]
I found these events have this issue: prophetofsierpinsk, waytoascension2

**Comment by ubertpendragon** [comment]
I think the one for recruiting Victoria Petran does it too if you first refuse

separatistspecialhire1  L342

**Comment by Regalis11** [comment]
This has now finally been addressed in our private repo (or at least, addressed in all of the problematic events that we could find, there could still be some we've missed). Fix coming up in the next unstable build and an upcoming release.
